pkg/app/app_test.go

```diff
@@ -217,6 +218,34 @@ func TestDeleteHostIsHandled(t *testing.T) {
 	}
 }
 
+func TestInfraConfigRequest(t *testing.T) {
+	controller := NewApp(&testInstanceManager{}, &testAccountManager{}, nil, nil, nil, "", nil, config.WebRTCConfig{STUNServers: []string{"foo.com:12345"}}, &config.Config{})
+	ts := httptest.NewServer(controller.Handler())
+	defer ts.Close()
+
+	res, _ := http.Get(ts.URL + "/v1/zones/foo/hosts/bar/infra_config")
+
+	expectedStatus := http.StatusOK
+	if res.StatusCode != expectedStatus {
+		t.Errorf("unexpected status code <<%d>>, want: %d", res.StatusCode, expectedStatus)
+	}
+
+	rBody, err := io.ReadAll(res.Body)
+	if err != nil {
+		t.Fatal(err)
+	}
+	var actualInfraConfig apiv1.InfraConfig
```

Comment:
nit: use `expected` and `got` variable names rather than `expectedInfraConfig` and `actualInfraConfig` avoiding verbose names.